ADAL/src/validation/ADAuthorityValidation.m

```diff
@@ -180,7 +187,14 @@ - (void)validateAuthority:(ADRequestParameters*)requestParams
     else
     {
         // Validate AAD authority
-        [self validateAADAuthority:authorityURL requestParams:requestParams completionBlock:completionBlock];
+        [self validateAADAuthority:authorityURL requestParams:requestParams completionBlock:^(BOOL validated, ADAuthenticationError *error)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 139 characters long line (which is greater than 120 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=objc%3ALineLength)